src/hydra/reqwest_client.rs

```diff
@@ -151,6 +151,10 @@ impl HydraClient for Client {
             Ok(r) => {
                 if r.status().is_success() {
                     Ok(())
+                } else if r.status() == 500 {
+                    // New hydra gives a 302 redirect to a page that ends up with a 500
+                    // Ideally we would not follow the redirect
```

Comment:
It seems to be possible to not follow redirect (https://docs.rs/reqwest/0.10.3/reqwest/redirect/struct.Policy.html#method.none). I think it would be a bit better than returning `Ok` on `500`.
Also, we could ask Hydra people about this behavior.

Reply:
Oh nice, I didn't see that part of the `reqwest` API :+1: